types/defines/rpc.d.ts

```diff
@@ -195,22 +195,35 @@ declare module "cloudflare:workers" {
     webSocketError?(ws: WebSocket, error: unknown): void | Promise<void>;
   }
 
-  export type DurationLabel =
+  export type WorkflowDurationLabel =
```

Comment:
This is a breaking change—I assume this is being considered okay because Workflows is experimental?

Reply:
Yhep this is okay as we are still iterating on the types, we just thought that would be better to prefix everything with Workflow to not pollute the scope